src/components/Navigator.js

```diff
@@ -444,84 +453,15 @@ class Navigator extends React.Component {
-    const useStyles = makeStyles((theme) => ({
-      root: {
-        padding: '2px 4px',
-        display: 'flex',
-        alignItems: 'center',
-        width: 400,
-      },
-      viewRoot: {
-        padding: '2px 4px',
-        display: 'flex',
-        alignItems: 'center',
-        width: 200,
-      },
-      input: {
-        marginLeft: theme.spacing(1),
-        flex: 1,
-      },
-      iconButton: {
-        padding: 5,
-      },
-      divider: {
-        height: 28,
-        margin: 4,
-      },
-    }));
-    const classes = useStyles();
-    const fileName = this.getFileName(fileId);
-    if(fileId.viewId !== 0){
-      const viewType = this.state.filesList[fileId.sourceId].views[fileId.viewId].type;
-      return (
-        <Paper className={classes.viewRoot} style={(selected) ? {backgroundColor: 'rgba(0, 0, 0, 0.08)'} : {} } component="form" elevation={0}>
-          <IconButton>
-            {
-              (viewType === FILE_TYPE.CARD_VIEW) ? 
-              <AmpStoriesIcon/>:
-              (viewType === FILE_TYPE.TEXT_VIEW) ?
-             <TextFieldsIcon /> :
-              null
-            }
-          </IconButton>
-          <InputBase 
-            value={fileName}
-            className="file_list_input"
-            disabled={!(selected && this.state.renameSelected)}
-          />
-          <Divider className={classes.divider} orientation="vertical" />
-          <IconButton className={classes.iconButton} onClick={handleEditMenuClick}>
-            <MoreVertIcon fontSize="small" color="disabled"/>
-          </IconButton>
-        </Paper>
-      );
-    }else{
-      return (
-        <Paper className={classes.root} style={(selected) ? {backgroundColor: '#a3d2f7', border:"solid #3f51b5 thin"} : {} } component="form" elevation={0}>
-          <InputBase 
-            value={fileName}
-            className="file_list_input"
-            disabled={!(selected && this.state.renameSelected)}
-          />
-          {childViewsExist ? (open ? <ExpandLess /> : <ExpandMore />) : null}
-          <Divider className={classes.divider} orientation="vertical" />
-          <IconButton className={classes.iconButton} onClick={handleEditMenuClick}>
-            <MoreVertIcon fontSize="small" color="disabled"/>
-          </IconButton>
-        </Paper>
-      );
-    }
-  };
-
   handleFileListClick = (fileId) => {
     if (fileId === this.state.selectedFileId) {
+      handleSetCurrentOpenFileId({ sourceId:fileId, viewId: 0, viewType: FILE_TYPE.EMPTY}, {sourceName: this.state.filesList[fileId].name, viewName: ''});
       this.setState({
         selectedViewId: null,
         selectedFileOpen: !(this.state.selectedFileOpen)
       });
     }else{
-      if(handleSetCurrentOpenFileId({ sourceId:fileId, viewId: 0 })){
+      if(handleSetCurrentOpenFileId({ sourceId:fileId, viewId: 0, viewType: FILE_TYPE.EMPTY}, {sourceName: this.state.filesList[fileId].name, viewName: ''})){
```

Comment:
Having the constant I mentioned before could be helpful to populate the `viewName` here, but now that I think about it an empty string isn't that helpful, so we could leave it as is